modules/bookCollection.js

```diff
@@ -0,0 +1,54 @@
+class BookCollection {
+  constructor(bookArray) {
+    this.bookArray = bookArray;
+  }
+
+  displayBooks() {
+    const bookContainer = document.querySelector('.books-container');
+    bookContainer.innerHTML = '';
+    this.bookArray.forEach((book) => {
+      bookContainer.insertAdjacentHTML(
+        'beforeend',
+        `
+        <div class="book-details" id="book-${book.id}">
+          <p>"${book.title}" by ${book.author}</p> 
+          <button class="remove" type="button" id="remove-${book.id}">Remove</button>
+        </div>`,
+      );
+    });
+
+    if (this.bookArray.length !== 0) {
+      bookContainer.style.border = 'solid 3px #000000';
+    } else bookContainer.style.border = 'none';
+  }
+
+  storeInLocalBrowser() {
+    localStorage.setItem('bookCollectionArray', JSON.stringify(this.bookArray));
+  }
+
+  removeBook(book) {
+    this.bookArray.splice(book, 1);
+    this.storeInLocalBrowser();
+    this.bookArray.forEach((book, i) => {
+      book.id = i;
+    });
+    this.displayBooks();
+  }
```

Comment:
- [ ] Currently when deleting a book its instead the first book on the list that's being removed. Please kindly fix that

Reply:
Hi @m05e5 , 
Thank you for taking the time to review my project.
I appreciate it.
However, on this requirement, I would like to highlight that, when the `splice` method is called on the `this.bookArray`
it removes the book specified in it as a parameter of its callback function.
Thank you again for your time.
cheers
@Tshobohwa